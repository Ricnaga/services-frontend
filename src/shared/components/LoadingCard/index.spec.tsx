import { render } from '@testing-library/react';
import { LoadingCard } from '.';

describe('LOADING CARD COMPONENT', () => {
  it('should be able to render component', () => {
    const { container } = render(<LoadingCard />);

    expect(container).toMatchSnapshot();
  });
});
