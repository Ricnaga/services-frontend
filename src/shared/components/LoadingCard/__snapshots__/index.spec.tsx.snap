// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOADING CARD COMPONENT should be able to render component 1`] = `
<div>
  <div
    class="w-100 mx-auto card"
  >
    <div
      class="card-body"
    >
      <div
        class="placeholder-glow card-title h5"
      >
        <span
          class="w-100 mx-auto my-2 placeholder"
        />
      </div>
      <p
        class="placeholder-glow card-text"
      >
        <span
          class="w-100 my-2 placeholder"
        />
        <span
          class="w-100 my-2 placeholder"
        />
        <span
          class="w-100 my-2 placeholder"
        />
        <span
          class="w-100 my-2 placeholder"
        />
      </p>
      <div
        class="placeholder-glow card-footer"
      >
        <span
          class="w-100 my-2 placeholder"
        />
      </div>
    </div>
  </div>
</div>
`;
